advocacy_docs/edb-postgres-ai/console/estate/monitor_aws.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Monitoring AWS resources in EDB Postgres AI 
+navTitle: Cloud Hosted Databases - AWS resources
+description: How to monitor AWS resources in EDB Postgres AI Estate.
+deepToC: true
+---
+
+## Overview 
+
+Setting up the EDB Postgres® AI Console to monitor your RDS instances and S3 buckets on AWS involves adding a specific policiy and role in AWS. Once these are configured, you need to enter the role ARN of the newly created role into the **Cloud Hosted Databases** wizard, accessible via the **Estate** page in the EDB Postgres AI Console. 
+
+Using this role ARN and a custom policy, the EDB Postgres AI server will have access to the RDS and S3 information in your AWS account. 
+
+After configuring the wizard with the role ARN, you will see the selected AWS resources (RDS instances and/or S3 buckets) in the chosen AWS regions on your **Estate** page in the **Cloud Hosted Databases** section.
+
+## Setting up monitoring of AWS resources in EDB Postgres AI Estate
+
+### Starting the Cloud Hosted Databases wizard
+
+1. Log into **EDB Postgres AI Console**. 
+
+2. Scroll down to the **Cloud Hosted Databases** section, select the **Manage Access** button, and choose your project.
+
+3. The **Cloud Hosted Databases** wizard shows **Step 1 - Create custom policy**
+
+### Creating the AWS custom policy
+
+4. Log into your AWS account with the RDS instances and S3 buckets you want to monitor.
+
+5. Navigate to IAM, and in the navigation pane on the left side of the AWS console, select **Policies**.
+
+6. On the **Policies** dashboard page, select the **Create policy**. button
+
+7. In the **Policy editor** section, choose the JSON option.
+
+8. Type or paste the following JSON policy document into the JSON editor:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": [
+        "rds:DescribeDBInstances",
+        "s3:ListAllMyBuckets",
+        "rds:DescribeDBClusters"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+9. Give the policy a name, for instance, **biganimal-beacon-addon-policy**. This policy allows EDB Postgres AI server to query metadata of your AWS RDS and S3 services.
+
+
+### Creating the AWS role
+
+10. In the wizard, select the **Next:Create a Role** button. The wizard should now show **Step 2 - Create a Role**.
```

Comment:
Also, you should really point out that these instructions are shown on the screen. Otherwise if you are following the docs *and* using the software your seeing two very similar sets of instructions.

Reply:
Oh yes and you need to make clear when you are talking about the EDB Postgres AI Console UI and the AWS Console UI.

Reply:
Ok, I think I kept them clear this next draft.